lib/cinch/plugins/word_game.rb

```diff
@@ -129,7 +129,7 @@ def start(m)
       if @game
```

Comment:
I can iterate over my branch, and once it's up to snuff I can squash the branch and resubmit.